packages/react-core/src/components/Dropdown/InternalDropdownItem.tsx

```diff
@@ -45,6 +45,8 @@ export interface InternalDropdownItemProps extends React.HTMLProps<HTMLAnchorEle
   customChild?: React.ReactNode;
   /** Flag indicating if hitting enter on an item also triggers an arrow down key press */
   enterTriggersArrowDown?: boolean;
+  /** Flag indicating if the item is a router link */
+  isRouterLink?: boolean;
```

Comment:
Could this prop just be isLink? Don't see "router" as adding value, considering you could add an HTML anchor instead.